### PR TITLE
Python: Add change note for unified parser.

### PR DESCRIPTION
**Pull request by markshannon**

### CODE REVIEW

File: change-notes/1.20/analysis-python.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Removes false positives seen when using Python 3.6, but not when using earlier v
 
  ## Changes to code extraction
 
- * *Series of bullet points*
+ * The extractor now parses all Python code from a single unified grammar. This means that almost all Python code will be successfully parsed, even if mutually incompatible Python code is present in the same project.
```
Comment (felicity-semmle) [inline change-notes/1.20/analysis-python.md]: It's not clear to me from reading this what this means for a user who's been struggling to get LGTM to analyze their project using the correct version of Python. Does this mean that they no longer need to worry about trying to set a version using the `lgtm.yml` file? Or is this still important to get the best results? I got the impression that this was a really important change, so we should make sure that people understand what it means for them.

Comment (markshannon) [reply change-notes/1.20/analysis-python.md]: The important thing to note is that the change is to the parser. If we use the wrong version of the standard library, we may still get false positives. So, correct configuration is still important.

For users who were confused by lots of syntax errors, the change to the alert message to include the interpreter version was probably more important.
What this does do, is enable us to parse projects with mutually incompatible Python code, such as ANTLR. For those projects there is no "correct" configuration.

Also, this change will also enable us to analyze code that is for a newer version of Python than on the LGTM workers. I've added that to the change note.

**Comment by felicity-semmle** [comment]
Thanks for the updates to the text. It's much clearer to me now.

**Comment by felicity-semmle** [comment]
@taus-semmle - are you happy for me to merge this?

**Comment by taus-semmle** [comment]
Sure, go right ahead! Thanks for pinging me (this PR didn't turn up in my list of PRs because I wasn't assigned to it).

**Pull request merged**